lighthouse-core/audits/seo/font-size.js

```diff
@@ -265,8 +279,9 @@ class FontSize extends Audit {
       });
     }
 
+    const percentFormatted = (percentageOfPassingText / 100).toFixed(4);
```

Comment:
wait hang on, triple sorry for this I seem to be all over the place with this PR 😭 

isn't this already going to be a string with 4 sigFigs before it ever gets to the i18n bit so all of what I've said is going to be moot anyhow?

Reply:
So I make it guaranteed to be 4 digits only, but not necessarily 4 sigfigs e.g. 0.0318 -> 3.18% which is 2 decimal places after the . but is only 3 sig figs.  So I guess maybe I should rename `percent4sigfig` to `percent2trailing` because it will clip off any fraction with more than 2 decimal places.  Technically I don't think I need to format this to 4 digits here since Intl will do that with the 4 max digits anyway.

Reply:
OK how's about this.

The `(percentageOfPassingText / 100).toFixed(4)` is super clear to me. Will i18n lib avoid a custom formatting option entirely if we can just pass that directly as just `'percent'` styled?